api/hooks/discordBot/commands/sdtd/status.js

```diff
@@ -53,17 +53,27 @@ class Status extends Commando.Command {
       onlinePlayersStringList += `${he.decode(player.name)}, `;
     });
 
+    let bloodMoonDay;
 
-    let nextHorde = (Math.trunc(serverInfo.stats.gametime.days / 7) + 1) * 7;
-    const daysUntilHorde = nextHorde - serverInfo.stats.gametime.days;
+    try{
+      bloodMoonDay = await sails.helpers.sdtdApi.executeConsoleCommand(
+        SdtdServer.getAPIConfig(sdtdServer),
+        `ggs BloodMoonDay`
+      );
+    } catch (error) {
+      sails.log.warn(`Hook - discordBot:status - ${error}`);
+      sails.log.error(error);
+    }
 
+    let nextHorde = parseInt(bloodMoonDay.result.split(` = `)[1]);
+    const daysUntilHorde = nextHorde - serverInfo.stats.gametime.days;
 
     let embed = new this.client.customEmbed();
 
     embed.setTitle(`${serverInfo.name} - status`)
       .addField('FPS', `${fps}`, true)
       .addField(`Gametime`, `${serverInfo.stats.gametime.days} days ${serverInfo.stats.gametime.hours} hours ${serverInfo.stats.gametime.minutes} minutes
-Next horde in ${daysUntilHorde} days`, true)
+        Next horde in ${daysUntilHorde} days`, true)
```

Comment:
did you intentionally add a new line here? it looks like it was one long line before. Discord probably won't care, but might be worth merging it back to one long line again

Reply:
The new line was there or else you'd see the carriage return "added" at the end of line 75. I think I added spaces for readability, but it isn't rendered in Discord. It is was a new line so that the next horde shows on a different line as it does here: 
![image](https://user-images.githubusercontent.com/31856462/95678100-5c83ad00-0b98-11eb-929a-2def70c8e330.png)